packages/session/src/sessions.ts

```diff
@@ -170,7 +170,9 @@ export class Sessions implements ISessions {
     const runningKernel = await this._kernels.get(kernelId);
     if (runningKernel) {
       runningKernel.disposed.connect(() => {
-        this.shutdown(sessionId);
+        // eslint-disable-next-line no-console
+        console.log('Session', sessionId, 'shut down');
+        // this.shutdown(sessionId);
```

Comment:
Just a note for our later selves, removing `this.shutdown(sessionId)` seems to be one way to make the "No Kernel on restart" issue go away.

Reply:
Yes, but we should also handle shutting down via the kernel list:

![image](https://github.com/user-attachments/assets/86f44471-6030-4681-9ab1-8487f981afdd)

I pushed a change in https://github.com/jupyterlite/jupyterlite/pull/1538/commits/f6beaa7cb3a2560626ccd1b1bb8614511e9cc352 to handle this.